.gitignore

```diff
@@ -1 +1 @@
-/samples/*
\ No newline at end of file
+/samples/*
```

Comment:
I think because some project I worked with insisted that JS users add to their global ignore files rather than allowing a `.gitignore` to be added to this person's repo, I may have enabled it myself and missed `node_modules` here. Yes, very good to do so! I've added a commit.